google-cloud-firestore/src/main/java/com/google/cloud/firestore/BulkWriter.java

```diff
@@ -167,24 +168,22 @@ enum OperationType {
   private final RateLimiter rateLimiter;
 
   /**
-   * The number of pending operations enqueued on this BulkWriter instance. An operation is
+   * The number of in-flight operations enqueued on this BulkWriter instance. An operation is
    * considered pending if BulkWriter has sent it via RPC and is awaiting the result.
```

Comment:
`considered pending if BulkWriter` should this be changed to "in-flight" as well?